src/bot/commands/correction.ts

```diff
@@ -102,9 +103,7 @@ export default class CorrectionCommand extends Command {
           return resolve(joke);
         }
 
-        question.channel
-          .send("Aucune blague n'a été trouvée, veuillez réessayer !")
-          .then((m) => setTimeout(() => m.deletable && m.delete().catch(() => null), 5000));
+        question.channel.send("Aucune blague n'a été trouvée, veuillez réessayer !").then(tDelete(5000));
```

Comment:
Est-ce vraiment utile de laisser se message sachant qu'y en a un déjà dans la fonction **this.findJoke()** ?